precompile/living-assets/src/tests.rs

```diff
@@ -2,96 +2,74 @@ use super::*;
 use evm::ExitRevert;
 use helpers::*;
 use sp_core::H160;
-use sp_runtime::{DispatchError, DispatchResult};
 
 type CollectionId = u64;
 type AccountId = H160;
 type AddressMapping = pallet_evm::IdentityAddressMapping;
 
-const CREATE_COLLECTION_0_OWNER_B7469C43535C826E29C30D25A9F3A035759CF132: &str = "1eaf25160000000000000000000000000000000000000000000000000000000000000000000000000000000000000000b7469c43535c826e29c30d25a9f3a035759cf132";
 const OWNER_OF_COLLECTION_0: &str =
 	"fb34ae530000000000000000000000000000000000000000000000000000000000000000";
+const CREATE_COLLECTION2: &str = "647f1a9c";
```

Comment:
I suggest to rename to CREATE_COLLECTION, removing the final '2'. Otherwise, by comparison with OWNER_OF_COLLECTION_0, it looks like one was trying to create a collection with id = 2.